src/main/fc/fc_msp.c

```diff
@@ -772,7 +772,11 @@ static bool mspCommonProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst, mspPostProce
         sbufWriteU16(dst, osdConfig()->cap_alarm);
         sbufWriteU16(dst, osdConfig()->time_alarm);
         sbufWriteU16(dst, osdConfig()->alt_alarm);
-        for (int i = 0; i < OSD_ITEM_COUNT; i++) {
+        int i;
+        for (i = 0; i < OSD_STAT_COUNT; i++ ) {
```

Comment:
No need to predeclare the loop variable, correct way for Betaflight is `for (int i = 0; ...`.